<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertData</class>
 <widget class="QDialog" name="insertData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>10</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть код товару</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Kod_Tovary">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>60</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть назву товару</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_Tovary">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>60</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть ціну товару</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Price_Tovary">
   <property name="geometry">
    <rect>
     <x>277</x>
     <y>60</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Kilkist_Tovary">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>60</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>407</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть кількість товару</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Termin_Pridatnosti">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>557</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть термін придатності</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Postavchik_Tovary">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>60</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>10</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Введіть поставника товару</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Dobaviti">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>100</y>
     <width>190</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавити</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
